### DISCUSSION TITLE
Welcome to SSRFPwned Discussions!

### DISCUSSION THREAD

**Discussion by blackhatethicalhacking** [blackhatethicalhacking/SSRFPwned#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JayaramanGanesh** [comment]
./SSRFPwned.sh: line 17: lolcat: command not found
./SSRFPwned.sh: line 20: lolcat: command not found
./SSRFPwned.sh: line 27: lolcat: command not found
Enter target domain: google.com
./SSRFPwned.sh: line 37: lolcat: command not found
./SSRFPwned.sh: line 38: waybackurls: command not found
./SSRFPwned.sh: line 40: lolcat: command not found
./SSRFPwned.sh: line 44: lolcat: command not found
./SSRFPwned.sh: line 47: lolcat: command not found
./SSRFPwned.sh: line 50: lolcat: command not found
./SSRFPwned.sh: line 53: lolcat: command not found
./SSRFPwned.sh: line 56: lolcat: command not found
./SSRFPwned.sh: line 61: lolcat: command not found
./SSRFPwned.sh: line 62: gf: command not found
Usage: httpx [OPTIONS] URL

Error: Missing argument 'URL'.
./SSRFPwned.sh: line 64: lolcat: command not found
./SSRFPwned.sh: line 68: lolcat: command not found
./SSRFPwned.sh: line 73: lolcat: command not found
./SSRFPwned.sh: line 98: google.com-20230627110411/vulnerable.txt: No such file or directory
./SSRFPwned.sh: line 99: lolcat: command not found


i got this error message ,what i do next

**Comment by blackhatethicalhacking** [comment]
Install the requirements first.

**Comment by JayaramanGanesh** [comment]
url format ?
